shared/insideResinOS/2.0.md

```diff
@@ -0,0 +1,154 @@
+# What is ResinOS
+
+* [Introduction](#introduction)
+* [Variants of ResinOS](#variants-of-resinos)
+  * [Dev vs. Prod images](#dev-vs-prod-images)
+  * [Standalone ResinOS](#standalone-resinos)
+* [ResinOS Components](#resinos-components)
+* [Stateless and Read-Only rootFS](#stateless-and-read-only-rootfs)
+* [Image Partition Layout](#image-partition-layout)
+* [OS Yocto Composition](#os-yocto-composition)
+
+## Introduction
+
+ResinOS is an operating system optimised for running Docker containers on embedded devices, with an emphasis on reliability over long periods of operation, as well as a productive developer workflow inspired by the lessons learned while building resin.io.
+
+The core insight behind resinOS is that Linux Containers offer, for the first time, a practical path to using virtualisation on embedded devices. VMs and hypervisors have lead to huge leaps in productivity and automation for cloud deployments, but their abstraction of hardware as well as their resource overhead and lack of hardware support means that they are out of the question for embedded scenarios. With OS-level virtualisation as implemented for Linux Containers, both those objections are lifted for Linux devices, of which there are many in the Internet of Things.
+
+ResinOS is an operating system built for easy portability to multiple device types (via the [Yocto framework](https://www.yoctoproject.org/)) and optimised for Linux Containers, and Docker in particular. There are many decisions, large and small, we have made to enable that vision, which are present throughout our architecture.
+
+The first version of ResinOS was developed as part of resin.io, and has run on thousands of embedded devices, deployed in many different contexts for several years. ResinOS v2 represents the combination of the learnings we extracted over those years, as well as our determination to make ResinOS a first-class open source project, able to run as an independent operating system, for any context where embedded devices and containers intersect.
```

Comment:
ResinOS should be lowercase resinOS throughout (except at the start of a sentence)